public/js/components/ButtonCard.vue

```diff
@@ -1,9 +1,12 @@
 <template>
   <div>
-    <button class="action-button" @click="onClick">
+    <button
+      class="action-button"
+      @click="onClick"
+      v-bind:aria-label="ariaLabel"
```

Comment:
This is counterproductive. The button already has an accessible name in the form of its button text. Providing an `aria-label` attribute overrides this when announcing the button in screen readers. Also, no component in our code base passed `ariaLabel` to a `ButtonCard` so this effectively _removes_ the button’s accessible text.

Why did you add this? Was it due to Axe reporting a violation?

Reply:
Hey @kleinfreund, yep, that's exactly why I added it. Here's the violation that Axe reported: 

![Screen Shot 2019-08-04 at 9 40 46 AM](https://user-images.githubusercontent.com/24294676/62425020-f51cc400-b69b-11e9-9ba1-a29cd1b9b3fc.png)

Here's the Axe documentation it refers to help fix the violation: https://dequeuniversity.com/rules/axe/3.1/button-name?application=axeAPI

Using the title attribute may be the simplest solution here:

![Screen Shot 2019-08-04 at 9 43 05 AM](https://user-images.githubusercontent.com/24294676/62425056-6f4d4880-b69c-11e9-8a5c-d945ffe2fd2e.png)

Axe doesn't report any violations with this update - let me know what you think.

Reply:
We could even add a prop for the title so we could be a bit more specific on the button title depending on how/where we're using the component - just a thought.

Reply:
Ah, then it’s very straightforward: In your tests for the ButtonCard, you want to provide a `buttonText` property on the `tile` object. Then, the button text won’t be empty. :)

Reply:
Lol oh I gotcha. Thanks man! Pushing your requested changes now.